nixos/modules/services/databases/openldap.nix

```diff
@@ -77,58 +135,101 @@ in
         example = [ "ldaps:///" ];
       };
 
+      settings = mkOption {
+        type = ldapAttrsType;
+        description = "Configuration for OpenLDAP, in OLC format";
+        example = lib.literalExample ''
+          {
+            attrs.olcLogLevel = [ "stats" ];
+            children = {
+              "cn=schema".includes = [
+                 "\${pkgs.openldap}/etc/schema/core.ldif"
+                 "\${pkgs.openldap}/etc/schema/cosine.ldif"
+                 "\${pkgs.openldap}/etc/schema/inetorgperson.ldif"
+              ];
+              "olcDatabase={-1}frontend" = {
+                attrs = {
+                  objectClass = "olcDatabaseConfig";
+                  olcDatabase = "{-1}frontend";
+                  olcAccess = [ "{0}to * by dn.exact=uidNumber=0+gidNumber=0,cn=peercred,cn=external,cn=auth manage stop by * none stop" ];
+                };
+              };
+              "olcDatabase={0}config" = {
+                attrs = {
+                  objectClass = "olcDatabaseConfig";
+                  olcDatabase = "{0}config";
+                  olcAccess = [ "{0}to * by * none break" ];
+                };
+              };
```

Comment:
Maybe some schema example would be nice here: https://github.com/Mic92/dotfiles/commit/666cc1d959c349defa6dab47e3077098e3ab32c2#diff-950b68ce0c8827469e259e1d33cd7d96R207

Reply:
Sure. Though the way I had intended that to work was: `settings.children."cn=schema".children."cn={2}postfix".attrs = ...;`